### ISSUE TITLE
UABEA keeps closing itself every time I batch import and export texture

### ISSUE THREAD

**Issue by TXV123** [nesrak1/UABEA#218]
I've been recently having an issue trying to batch import textures on BBCR and it was working fine prior to the issue but then out of nowhere the issue started. And it doesn't even show an error message, it just closes. I don't know what's causing it or how to fix it so if anyone out there to help, I would appreciate it.

**Comment by nesrak1** [comment]
Do you have a uabeacrash.log file? Someone else reported this in the Discord but the crash log didn't exist and I couldn't fix their problem.

**Comment by TXV123** [comment]
I do. And here's what the file said when I opened it up with notepad: 

System.DllNotFoundException: Unable to load DLL 'textoolwrap' or one of its dependencies: The specified module could not be found. (0x8007007E)
   at TexturePlugin.PInvoke.EncodeByPVRTexLib(IntPtr data, IntPtr buf, Int32 mode, Int32 level, UInt32 width, UInt32 height)
   at TexturePlugin.TextureEncoderDecoder.EncodePVRTexLib(Byte[] data, Int32 width, Int32 height, TextureFormat format, Int32 quality)
   at TexturePlugin.TextureEncoderDecoder.Encode(Byte[] data, Int32 width, Int32 height, TextureFormat format, Int32 quality)
   at TexturePlugin.TextureImportExport.Import(String file, TextureFormat format, Int32& width, Int32& height)
   at TexturePlugin.ImportTextureOption.ImportTextures(Window win, List`1 batchInfos)
   at TexturePlugin.ImportTextureOption.ExecutePlugin(Window win, AssetWorkspace workspace, List`1 selection)
   at UABEAvalonia.PluginWindow.BtnOk_Click(Object sender, RoutedEventArgs e) in C:\Users\karse\Documents\GitReposLocal\UABEA\UABEAvalonia\PluginWindow.axaml.cs:line 59
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
   at Avalonia.Threading.JobRunner.JobWithArg.Avalonia.Threading.JobRunner.IJob.Run() in /_/src/Avalonia.Base/Threading/JobRunner.cs:line 241
   at Avalonia.Threading.JobRunner.RunJobs(Nullable`1 priority) in /_/src/Avalonia.Base/Threading/JobRunner.cs:line 38
   at Avalonia.Win32.Win32Platform.WndProc(IntPtr hWnd, UInt32 msg, IntPtr wParam, IntPtr lParam) in /_/src/Windows/Avalonia.Win32/Win32Platform.cs:line 287
   at Avalonia.Win32.Interop.UnmanagedMethods.DispatchMessage(MSG& lpmsg)
   at Avalonia.Win32.Win32Platform.RunLoop(CancellationToken cancellationToken) in /_/src/Windows/Avalonia.Win32/Win32Platform.cs:line 214
   at Avalonia.Threading.Dispatcher.MainLoop(CancellationToken cancellationToken) in /_/src/Avalonia.Base/Threading/Dispatcher.cs:line 61
   at Avalonia.Controls.ApplicationLifetimes.ClassicDesktopStyleApplicationLifetime.Start(String[] args) in /_/src/Avalonia.Controls/ApplicationLifetimes/ClassicDesktopStyleApplicationLifetime.cs:line 120
   at Avalonia.ClassicDesktopStyleApplicationLifetimeExtensions.StartWithClassicDesktopLifetime[T](T builder, String[] args, ShutdownMode shutdownMode) in /_/src/Avalonia.Controls/ApplicationLifetimes/ClassicDesktopStyleApplicationLifetime.cs:line 209
   at UABEAvalonia.Program.Main(String[] args) in C:\Users\[Censored]\Documents\GitReposLocal\UABEA\UABEAvalonia\Program.cs:line 52
   
(I only censored the user name.) 

If any of this will provide more info.
